app/about.js

```diff
@@ -0,0 +1,118 @@
+  }
+}
+//parsing the package.json file
+const pkg = JSON.parse((fs.readFileSync(PKG)));
+
+
+function toggleLicense() {
+  const license = document.querySelector("#license");
+  const main = document.querySelector("#main");
+  if (license.classList.contains("top")) {
+    license.classList.remove("top");
+    license.querySelector("button").textContent = "Näytä lisenssi";
+    license.setAttribute("aria-hidden", "true");
+    main.setAttribute("aria-hidden", "false");
+  } else {
+    license.classList.add("top");
+    license.querySelector("button").textContent = "Piilota lisenssi";
+    license.setAttribute("aria-hidden", "false");
+    main.setAttribute("aria-hidden", "true");
+  }
+}
+
+const licenseFile = fs.readFileSync(LICENSE).toString()
+pkg['licenseFile'] = licenseFile;
+
+const copyrights = [];
+for (const line of licenseFile.split('\n')) {
+    if (line.startsWith('Copyright (c) ')) {
+        copyrights.push(line.replace('Copyright (c)', '\u00a9'));
+    } else {
+        break;
+    }
+}
+
+render(
+  <div id='about-dialog'>
+    <header>
+      <img alt="" height="35" src="resources/sensotrend.svg"></img>
+    </header>
+    <article id="main" aria-hidden="false">
+    <h2>{pkg.description} <small>v{pkg.version}</small></h2>
+    <section id="ce">
+      <img alt="CE" src="resources/ce.svg" />
+      <p>{i18n.t('Sensotrend Uploader on luokan I lääkinnällinen laite.')}</p>
+      <p>{i18n.t('Ks.')} <a
+        href="https://www.sensotrend.fi/connect/static/media/DECO-T5-DC-MDD_EU_Declaration_of_Conformity_-_Sensotrend_Uploader_-_signed.17445413.pdf">{i18n.t('Vaatimustenmukaisuusvakuutus')}</a>.</p>
+    </section>
+    <section id="device">
+      <img alt={i18n.t('Unique Device Identification')} title={i18n.t('Unique Device Identification')} src="resources/udi.png" />
+      <p>UDI-DI: D-FIMF000000500SU000001QT</p>
+    </section>
+    <section id="date">
+      <img alt={i18n.t('Manifacturing date')} title={i18n.t('Manifacturing date')} src="resources/manufacturingDate.png" />
+      {BuildTime}
+    </section>
+    <section id="manufacturer">
+      <img alt={i18n.t('Manifacturer')} title={i18n.t('Manifacturer')} src="resources/manufacturer.png" />
+      <address>Sensotrend Oy<br />
+        Tampellan esplanadi 19 A 55<br />
+        33180 Tampere, Finland<br />
+        <a href={i18n.t('mailto:tuki@sensotrend.com')}>{i18n.t('tuki@sensotrend.com')}</a>
+      </address>
+    </section>
+    <section id="instructions">
+      <img alt={i18n.t('e-instructions')} title={i18n.t('e-instructions')} src="resources/e-instruction.png" />
+      <p><a href={i18n.t('https://www.sensotrend.fi/connect/instructions/uploaders/?l=en')}>{i18n.t('Instructions')}</a></p>
+    </section>
+    <section id="info">
+      <img alt={i18n.t('Additional info')} title={i18n.t('Additional info')} src="resources/website.png" />
+      <p><a href={i18n.t('https://www.sensotrend.com/uploader.html')}>{i18n.t('https://www.sensotrend.com/uploader')}</a></p>
```

Comment:
Onko tässä nyt kaksi linkkiä peräkkäin? Ihan en pelkästä koodista hahmota, miltä tämä näyttää, mutta itse toiveena olisi yksi oikean kielinen linkki.

Reply:
Siinä on osoite mihin linkki vie, ja sen jälkeen oleva teksti on minkä käyttäjä näkee, vanhemmassa versiossa oli tuo www.sensotrend.fi/uploader mikä näkyi käyttäjille, niin pidin sen samanlaisena, mutta se kääntää käyttäjille näkyvän tekstin jotta se olisi samanlainen kuin aikaisempi. Olisikohan sille olemassa jokin parempi tapa mikä teksti näkyisi käyttäjille? esim. Uploader kotisivut?

Reply:
Elikkäs jos kieli on suomeksi vie linkki www.sensotrend.fi/uploader, mutta jos kieli on joksikin muuksi kuin suomeksi niin se vie www.sensotrend.com/uploader.html